src/tmpfiles/tmpfiles.c

```diff
@@ -2821,10 +2819,12 @@ int main(int argc, char *argv[]) {
 
         mac_selinux_finish();
 
-        if (r < 0)
+        if (r < 0 || r2 == -ENOMEM)
```

Comment:
I think it would make sense to list EMFILE and ENFILE here too... maybe wrap this in a macro like the existing ERRNO_IS_DISCONNECT()? i.e. ERRNO_IS_RESOURCE() or ERRNO_IS_EXHAUSTED() or ERRNO_IS_RES_EXHAUSTED() or so, which lists at least these three errnos?